Mathlib/LinearAlgebra/FreeModule/Finite/Rank.lean

```diff
@@ -60,17 +60,18 @@ variable [AddCommGroup N] [Module R N] [Module.Finite R N]
 
 /-- The rank of a finite module is finite. -/
 theorem rank_lt_aleph0 : Module.rank R M < ℵ₀ := by
-  dsimp only [Module.rank]
+  clear! N
```

Comment:
What's going on here?

Reply:
(answer: the single french quote syntax is pulling in `N` even though it's not used! I changed the spelling to avoid that.